udata/core/organization/factories.py

```diff
@@ -13,8 +13,8 @@ class OrganizationFactory(MongoEngineFactory):
     class Meta:
         model = models.Organization
 
-    name = factory.LazyAttribute(lambda o: faker.sentence())
-    description = factory.LazyAttribute(lambda o: faker.text())
+    name = factory.LazyAttribute(lambda o: faker.text())
```

Comment:
I don't think so. `faker.text()` output an entire paragraph whereas `faker.sentence()` output a single sentence,  which is more consistent with a title or a name.